src/libraries/intellisense.targets

```diff
@@ -75,6 +79,7 @@
     <ProjectAssetsCacheFile Condition="'$(ProjectAssetsCacheFile)' == ''">$(IntermediateOutputPath)$(MSBuildProjectName).assets.cache</ProjectAssetsCacheFile>
     <ProjectAssetsCacheFile>$([MSBuild]::NormalizePath($(MSBuildProjectDirectory), $(ProjectAssetsCacheFile)))</ProjectAssetsCacheFile>
     <NuGetTargetMoniker>$(TargetFrameworkMoniker)</NuGetTargetMoniker>
+    <TargetPath>$(OutputPath)$(TargetFileName)</TargetPath>
```

Comment:
Why is this needed?

Reply:
TargetPath is the final path to the test dll, it is calculated using the output path in .targets file. After we override the output path we need to update the targetPath as well.

Reply:
You need to add comments here describing why you do this sort of thing.